hack/Vagrantfile.netbsd

```diff
@@ -0,0 +1,31 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "generic/netbsd9"
+  config.vm.boot_timeout = 900
+  config.vm.synced_folder ".", "/vagrant", type: "rsync"
+  config.ssh.keep_alive = true
+
+  config.vm.provision "init", type: "shell", run: "once" do |sh|
+    sh.inline = <<~SHELL
+      mkdir -p /var/tmp
+      chmod 1777 /var/tmp
+
+      pkgin -y install git mozilla-rootcerts
+      mozilla-rootcerts install
+
+      ftp https://go.dev/dl/go1.23.3.netbsd-amd64.tar.gz
+      tar -C /var/tmp -xzf go1.23.3.netbsd-amd64.tar.gz
+
+      cat << 'EOF' > /usr/bin/go-wrapper
+      #!/bin/sh
+      export TMPDIR="/var/tmp"
```

Comment:
needs to override TMPDIR otherwise tests fail with no space left error because `/tmp` partition on netbsd vagrant box doesn't have enough space (512MB).